### PR TITLE
Implement international relocation payment (IRP) variables

### PR DESCRIPTION
**Pull request by Sarah-DfE**
### Trello card

https://trello.com/c/RyNnbnnt/6274-implement-international-relocation-payment-irp-variables

### Context
implement the international relocation payment variables on the following pages
/non-uk-teachers/get-an-international-relocation-payment
/non-uk-teachers/teach-in-england-if-you-trained-overseas
/non-uk-teachers/ukraine

### CODE REVIEW

**Comment by Sarah-DfE** [comment]
Ready for review please @martyn-w @gemmadallmandfe

**Review by martyn-w — APPROVED** [review]
👍 LGTM (from tech perspective)

File: app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md [hunk 1]
```diff
@@ -50,9 +50,9 @@ If you are already a qualified teacher, you do not need to retrain to teach in E
 * to pass criminal and professional safeguarding checks (these will be organised by your employer)
 * [a visa or immigration status allowing you to work in the UK](/non-uk-teachers/visas-for-non-uk-teachers)
 
-## Get an international relocation payment (IRP) worth £10,000
+## Get an international relocation payment (IRP) worth $nonuk_internationalrelocationpayment_value$
 
-If you're interested in moving to England to teach languages or physics in an English secondary school, you could be eligible for a single payment of £10,000 from the UK government. [Learn more about the international relocation payment](/non-uk-teachers/get-an-international-relocation-payment).
+If you're interested in moving to England to teach languages or physics in an English secondary school, you could be eligible for a single payment of $nonuk_internationalrelocationpayment_value$ from the UK government. [Learn more about the international relocation payment](/non-uk-teachers/get-an-international-relocation-payment).
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md:55]: @Sarah-DfE Not sure `single payment` is correct now? Can you check with policy how this should be worded and raise a separate ticket and PR for this?

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**